### PR TITLE
Mono flavor

### PR DESCRIPTION
**Pull request by takaaki-fukunaga**
I have added mono flavor. Could you add these changes?

### CODE REVIEW

**Comment by takaaki-fukunaga** [comment]
I have renamed How-to-Use.md to emlinux-image-mono.md and revised it for only Mono. 

I would like to add some flavoers and add **Confirm operation** section for each document. It's annoying to maintain each document but it's good for a user to understand how to use each flavor.

**Review by hiromotai7 — CHANGES_REQUESTED** [review]
Could you add this commit into 8d1c1a2f4f436cb1cd268bd4e3b33179070b43a1 by git-rebase?

**Review by hiromotai7 — CHANGES_REQUESTED** [review]
Could you add this commit into 787467f1c43c62cb5deb88a19a9174139f441766?

File: recipes-core/images/emlinux-image-mono.bb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#
+# Gnome Linux image
+#   based on EMLinux base image
+#
+# SPDX-License-Identifier: MIT
+#
+
+require recipes-core/images/emlinux-image-base.bb
+
+DESCRIPTION = "Gnome target root filesystem"
```
Comment (hiromotai7) [inline recipes-core/images/emlinux-image-mono.bb]: I would be happy if you used "emlinux-image-flavorname.bb.sample"....

**Comment by hiromotai7** [comment]
@takaaki-fukunaga Thanks.
I have made some comments and would be happy to correct them.

Finally, if you run "git diff a5c1be78dc41 new-rev" and there is no difference between the current branch and new fixed branch, you can do "git push --force" the new branch as your/main branch. It means you do not need to make another merge request.

**Comment by takaaki-fukunaga** [comment]
@hiromotai7 
I have rebased the main branch and pushed the changes. Could you check it when you have time?

**Comment by hiromotai7** [comment]
@takaaki-fukunaga 
Thank you. but you missed "sign-off-by" in commit 0e485e9213cd17c19b9574d723cf6583deac53bf.
Could you fix it?

**Comment by hiromotai7** [comment]
@takaaki-fukunaga 
Thanks!

**Pull request merged**